prog/vm/host_nexus.rb

```diff
@@ -83,8 +83,7 @@ def before_run
         vm_host.update(**kwargs)
       when "LearnStorage"
         kwargs = {
-          total_storage_gib: st.exitval.fetch("total_storage_gib"),
```

Comment:
Some learn progs, like LearnNetwork, directly update the VM host within the prog itself. Others, like LearnMemory, simply return the learned value, allowing the parent to perform the update. When I asked @fdr about this inconsistency in the past, he said, "We're unsure which method is superior. Let's maintain both styles and decide in the future." LearnStorage introduced a new pattern as well. It creates StorageDevices but returns the total storage for the VmHost model. Therefore, if we run LearnStorage but don't utilize the returned value, it could lead to some inconsistencies. IMHO, we need to decide whether to persist data within the prog or outside of it. In this context, we could also consider moving the update of vm host total_storage_gib into the LearnStorage prog itself.

Reply:
Removed it.